Status: Alpine specific - disables compilation of stuff irrelevant to Alpine

--- a/dict/Makefile.am
+++ b/dict/Makefile.am
@@ -1,7 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-SUBDIRS = m4 src stardict-plugins tests pixmaps po help data doc \
-	debian-upstream mac_dist msvc_2008
+SUBDIRS = m4 src stardict-plugins tests pixmaps po help data doc
 ACLOCAL_AMFLAGS = -I m4
 
 EXTRA_DIST = \
